build/Targets/Imports.targets

```diff
@@ -498,4 +498,13 @@
     <AdditionalFiles Include="@(PublicAPI)" />
   </ItemGroup>
 
+  <!-- CPS doesn't show these items by default, but we want to show them. -->
```

Comment:
Well technically CPS does add these files as `<None>` items, but we explicitly disabled that feature. I imagine that this will be addressed by dotnet/project-system#113.